developer_docs/order_in_C.md

```diff
@@ -2,7 +2,7 @@
 
 ## Order  
 
-`order` is the parameter given the `numpy.array` in order to choose how the array is stored in memory, both  `Pyccel` supported orders are stored contiguously in memory, they differ in the order - the order of the values -.
+`order` is the parameter given the `numpy.array` in order to choose how the array is stored in memory, both  `Pyccel` supported orders are stored contiguously in memory, they differ in the order -the order by which the values are stored in memory.
```

Comment:
is there an order which is not `stored contiguously in memory` ?

Reply:
No, `K` and `A` orders interact somehow with the `order_f` and the `order_c` orders, but do not have their own specific behaviour.
I think that we should only focus on `order_f` and `order_c` for pyccel, that's why I did not mention the other `numpy` orders.
There is also the traditional `C` way of storing data -pointers and double pointers and triple pointers...- which is not contiguous, should I mention it in the docs?

Reply:
In numpy you can have an array which is neither C ordered, nor F ordered meaning it is non-contiguous in memory. But I don't think this is relevant to pyccel. In our case it is more about whether it is row ordered or column ordered so we know the most efficient way to index

Reply:
`K` and `A` are orders that can be specified on array creation. They are mapped to `C` and `F` under the hood. I think all arrays on contiguous but not all views (pointers)

Reply:
Oh yes! forgot about views